### ISSUE TITLE
Error message received when updating filters

### ISSUE THREAD

**Issue by Ringnut** [3ll3d00d/beqdesigner#216]
This error message was displayed when I tried to update filters from the 'Preferences' page.

Traceback (most recent call last): 
File "model\preferences.py", line 596, in updateBeq 
File "model\preferences.py", line 605, in __pull_beq 
File "site-packages\dulwich\porcelain.py", line 859, in pull 
File "site-packages\dulwich\client.py", line 403, in fetch 
File "site-packages\dulwich\client.py", line 389, in commit 
File "site-packages\dulwich\object_store.py", line 702, in add_thin_pack 
File "site-packages\dulwich\object_store.py", line 662, in _complete_thin_pack 
PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'E:/Jon/Documents/Calibration/BassEQ/Files/Movie BEQs\\.git\\objects\\pack\\pack-7c6be9ab71da4e548cd8c66f30124a33e1b6571c.pack'

**Comment by 3ll3d00d** [comment]
Windows is very picky about file access so this can be caused by any file in that directory being open in any process on the box. This, plus the fact I find this happens v intermittently on Windows, makes it extremely hard to fix.

So basically restart the app and try again!

**Comment by Ringnut** [comment]
Issue solved by removing duplicate folders.

**Comment by Ringnut** [comment]
Thanks Matt,

 

I tried shutting down and restarting and I tried logging off and restarting neither of which worked. I then discovered that there were duplicate folders created so I put them in a separate storage folder and then restarted. Going into ‘Preferences’ the ‘files loaded’ box reported no files. I then pressed the refresh button and it immediately repopulated with all the files. So all resolved and thank you for your help.

 

Thank you also for creating this programme, I never realised you were so talented! J I never cease to be amazed at how much you and others do for us in this addictive hobby of AV. 

 

Thanks again,

 

Jon
